### PR TITLE
fix(event-bridge): use inline role policy

### PR DESCRIPTION
**Pull request by cgeers**
from event bridge, removes the policy document, policy, and role policy attachment in favor of an inline role policy.

### CODE REVIEW

**Review by nkraemer-sysdig — COMMENTED** [review]
Looks good, but this will break the snippets currently shown in the UI right? Since they rely on the output of one module as the input to another?

File: modules/services/event-bridge/main.tf [hunk 1]
```diff
@@ -51,7 +51,7 @@ EOF
 # See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_event_target#cross-account-event-bus-target
 resource "aws_cloudwatch_event_target" "sysdig" {
   count      = var.is_organizational && !var.provision_management_account ? 0 : 1
-  depends_on = [aws_iam_role.event_bus_invoke_remote_event_bus]
+  depends_on = [aws_iam_role.event_bus_invoke_remote_event_bus, aws_cloudwatch_event_rule.sysdig]
```
[the remark this answers is not in the document]
Comment (cgeers) [reply modules/services/event-bridge/main.tf:54]: Why? To be clear. We've demonstrated that implicit dependencies don't behave exactly as we expect in all cases.

**Comment by cgeers** [comment]
> Looks good, but this will break the snippets currently shown in the UI right? Since they rely on the output of one module as the input to another?

https://sysdig.atlassian.net/browse/SSPROD-24317 is intended to address this.

**Review by nkraemer-sysdig — DISMISSED** [review]
LGTM but needs rebasing after the Org TFs

**Review by haresh-suresh — APPROVED** [review]

**Comment by cgeers** [comment]
rebased and preserved organizational `count` changes in #16

**Review by nkraemer-sysdig — APPROVED** [review]

**Pull request merged**